editor/sidebar/post-author/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import { PanelRow, withInstanceId } from 'components';
+import { Component } from 'element';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getEditedPostAttribute } from '../../selectors';
+import { editPost } from '../../actions';
+
+class PostAuthor extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			authors: [],
+		};
+	}
+
+	fetchAuthors() {
+		this.fetchAuthorsRequest = new wp.api.collections.Users().fetch( { data: {
+			roles: 'author,editor,administrator',
```

Comment:
>Good question and I answer with another question: Is is ok to set a "subscriber" as a post author?

Sorry, I had been thinking about this the other day and meant to respond: We should request users by capability, not role. Specifically the [`edit_posts`](https://codex.wordpress.org/Roles_and_Capabilities#edit_posts) capability. This is not currently supported by the [list endpoint](https://developer.wordpress.org/rest-api/reference/users/#list-users), however. Probably an API task worth exploring. I'll create an issue.